src/util/configHelpers.js

```diff
@@ -305,6 +305,23 @@ const validKey = (key, allKeys) => {
   return [isUniqueKey, { key }];
 };
 
+const validUserTypesForUserConfig = (includeForUserTypes, userTypesInUse) => {
```

Comment:
You have this function, but you don't use it anywhere.

Reply:
Updated user field validation to use this! If no user types are passed to this function, it sets the valid value to 

```
      userTypeConfig: {
        limitToUserTypeIds: false,
      },
```